### PR TITLE
Destop skeleton

### PR DESCRIPTION
**Pull request by SFarida**
The following were update: 
- the nav bar in the header to include the menu list
- the Intro section items were reordered to fit bigger screens
- the work section items were reordered to fit bigger screens
- the form as well was reordered
- the  footer list display was changed to grid

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,411 +1,383 @@
...
         <div class="works_heading">
           <h3>My Recent Works</h3>
         </div>
-        <hr class="underline" />
-        <div class="recent_works">
-          <div>
-            <img width="100%" src="img/works/imgPlaceholder.png" alt="Work" />
+        <div class="underline">
+          <!-- <hr class="underline" /> -->
+        </div>
```
Comment (Ugboaja-Uchechi) [inline index.html]: - [ ] Kindly remove all commented code to make your codebase neater and easier to read 🙏

File: index.html [hunk 2]
```diff
@@ -1,411 +1,383 @@
...
+          </p>
+          <ul class="stacks">
+            <li class="stack_item stack_item1">css</li>
+            <li class="stack_item stack_item2">html</li>
+            <li class="stack_item stack_item3">bootstrap</li>
+            <li class="stack_item stack_item4">Ruby</li>
+          </ul>
+          <button type="button" class="see_projects_btn btn_text">
+            <span class="btn_text">See Project</span>
+          </button>
```
Comment (Ugboaja-Uchechi) [inline index.html:103]: - [ ] Great job with your design 👏. Kindly make sure there is no `button` in your design until one of the card is hovered 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

- [ ] Kindly make sure when one of the cards are hovered on the `linear-gradient` is removed and your design looks as close as possible to your chosen Figma design 🙏 

image

File: index.css [hunk 3]
```diff
@@ -549,9 +566,11 @@ body {
 /** Skills */
 .skills {
   display: flex;
-  flex-direction: column;
-  align-items: center;
+  flex-direction: row;
   gap: 24px;
+  margin-top: 120px;
+  justify-content: space-around;
+  flex-wrap: wrap;
```
Comment (Ugboaja-Uchechi) [inline index.css:581]: - [ ] Kindly make sure the list of technologies are aligned correctly as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: index.css [hunk 4]
```diff
@@ -635,29 +652,26 @@ body {
 
 .form {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
```
Comment (Ugboaja-Uchechi) [inline index.css:663]: - [ ] Kindly make sure your text and form section are aligned correctly as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: index.css [hunk 5]
```diff
@@ -203,16 +221,15 @@ body {
...
+  border-top: 1px solid #212121;
+  width: 98%;
+  align-self: center;
+  margin-left: 2%;
 }
 
 .recent_works {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
```
Comment (Ugboaja-Uchechi) [inline index.css:232]: - [ ] Kindly adjust the spacing between your img and text to look as close as possible to your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: index.css [hunk 6]
```diff
@@ -43,39 +62,33 @@ body {
...
   align-items: flex-start;
-  padding: 120px 24px;
+  height: 100vh;
+}
+
+.intro_item1 {
+  order: 1;
+  margin: 20% 5% 0 5%;
+}
```
Comment (Ugboaja-Uchechi) [inline index.css:82]: - [ ] On mobile screen starting from `401px` your design doesn't match the mobile Figma design. Kindly fix this 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @SFarida,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 7]
```diff
@@ -537,37 +562,38 @@ body {
...
-  margin: 120px 24px 0 24px;
+  margin: 10px 5% 0 5%;
 
   /* no unit */
 }
 
 .multi_post {
-  width: 100%;
+  width: 40%;
 }
```
Comment (binhussen) [inline index.css:572]: - Kindly, stick to the Figma design as much as possible, the description of your recent work is centered make some Figma as your previous code reviewer suggested.👌👍



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

File: index.css [hunk 8]
```diff
@@ -635,29 +660,26 @@ body {
...
   gap: 24px;
-  flex: none;
   flex-grow: 0;
   align-self: center;
 }
 
 .form_item1 {
   order: 0;
-  margin-top: 120px;
 }
```
Comment (binhussen) [inline index.css]: - Kindly, stick to the figma design as match as possible, and align the text correctly as your previous code reviewer requested you.👌👌



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @SFarida , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- Kindly, stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1278&t=FtGxbkcwEURbVBw0-0) as match as possible.👌

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -1,411 +1,381 @@
...
-    </main>
-  </body>
-</html>
+    </section>
+    <div class="form">
+      <div class="form_item form_item1">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd
+          like to chat please get in touch.
+        </p>
```
Comment (abdulhamiid) [inline index.html:334]: - [OPTIONAL] please consider using the same text font as the Figma design in this section to match the template as stated in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md).

Your design 👇 
image

Figma design 👇 
image

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @SFarida  
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 9 above)
Comment (SFarida) [reply index.html:334]: Hello @abdulhamiid  Thank you for the review. the font weight is the same, just that when applied it looks like this.

**Pull request merged**